config/run-config.yaml

```diff
@@ -1,12 +1,14 @@
 projects:
   audacity:
-    git_url: "https://github.com/audacity/audacity.git"
+    git_url: "https://github.com/audacity/audacity"
+    branch: "master"
```

Comment:
I added the branch because the downloader needs it to find a file. Not all repositories have `main` as the main branch and so we could work on any branch of a project (not sure if that's useful yet 😅 ).